src/components/Input/FileInput.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ChangeEvent } from 'react';
+import Input from './Input';
+
+type FileInputProps = {
+  onChange: (files: FileList | null) => void;
+  type: 'file';
+};
+
+class FileInput extends React.Component<FileInputProps> {
+  handleChange = (changeEvent: ChangeEvent) => {
+    const files = (changeEvent.target as HTMLInputElement).files;
```

Comment:
Use type generic here to avoid casting
```suggestion
  handleChange = (changeEvent: ChangeEvent<HTMLInputElement>) => {
    const files = changeEvent.target.files;
```